docs/pt-BR/concepts/cli.mdx

```diff
@@ -3,6 +3,7 @@ title: CLI
 description: Aprenda a usar o CLI do CrewAI para interagir com o CrewAI.
 icon: terminal
 ---
+<Warning>A partir da versão 0.135.0, a plataforma CrewAI Enterprise iniciou um processo de migração de seu provedor de login. Como resultado, o fluxo de autenticação via CLI foi atualizado. Usuários que utlizam o Google para fazer login, ou que criaram conta após 3 de julho de 2025 não poderão fazer login com versões anteriores da biblioteca `crewai`.</Warning>
```

Comment:
it wont be 0.135.0. will reply the next v

Reply:
fixed the docs, thanks! next version is v0.140.0